store/tikv/2pc.go

```diff
@@ -112,6 +112,8 @@ type twoPhaseCommitter struct {
 	regionTxnSize map[uint64]int
 	// Used by pessimistic transaction and large transaction.
 	ttlManager
+	// Used for select for update in ms, 0 means always wait, 1 means nowait
+	lockWaitTime uint64
```

Comment:
Why should `lockWaitTime` be a struct field in twoPhaseCommitter?

```
begin;
select xxx for update no wait
select yyy for update  // The committer is still there but `lockWaitTime` change?
commit;
```

How about pass `lockWaitTime` as an argument?

Reply:
`pessimisticLockSingleBatch` and other  proc functions from `getProcFuncByType` have the same declaration,  seems need more change to pass it as a variable into `NewRequest` call

Reply:
I did a refactoring and `twoPhaseCommitAction` can carry more information now. You can see how `actionPessimisticLock` is done on the master branch.

Never mind, as we can also merge the PR and refactor later.

Reply:
Roger, I've learned the `action` refactoration, this is refactored passing time param as parameter not member of `twoPhaseCommitter`